src/js/components/tasks/capture/directories.tsx

```diff
@@ -0,0 +1,89 @@
+import Radio from '@salesforce/design-system-react/components/radio';
+import RadioGroup from '@salesforce/design-system-react/components/radio-group';
+import i18n from 'i18next';
+import React from 'react';
+import { Trans } from 'react-i18next';
+
+import { CommitData } from '@/components/tasks/capture';
+import { TargetDirectories } from '@/store/orgs/reducer';
+
+interface Props {
+  directories: TargetDirectories;
+  inputs: CommitData;
+  errors: {
+    [key: string]: string;
+  };
+  handleInputChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+}
+
+const TargetDirectoriesForm = ({
+  directories,
+  inputs,
+  errors,
+  handleInputChange,
+}: Props) => {
+  const headings = {
+    source: i18n.t('Package Directories'),
+    pre: i18n.t('Pre-Install Directories'),
+    post: i18n.t('Post-Install Directories'),
+    config: i18n.t('Optional Configuration Directories'),
+  };
+  const keys = [
+    'source' as 'source',
+    'pre' as 'pre',
+    'post' as 'post',
+    'config' as 'config',
+  ];
+  const orderedDirectories = new Map<
+    'source' | 'pre' | 'post' | 'config',
+    string[]
+  >();
+  keys.forEach((key) => {
+    if (directories[key]?.length) {
+      orderedDirectories.set(key, directories[key] as string[]);
+    }
+  });
+
+  return (
+    <form className="slds-form slds-p-around_large">
+      <div className="slds-form-element__help slds-p-bottom_small">
+        <Trans i18nKey="targetDirectoryInfo">
+          Here’s an explanation of what each type of directory is. The Package
+          Directories are this and the Pre-Install and Post-Install Directories
+          are different. The Optional Configuration Directories are something
+          else entirely.
```

Comment:
@davisagli I'd appreciate your suggestions for some explanatory content here.

Reply:
@jgerigmeyer gah sorry, I wrote this a few days ago but failed to submit my review.

Package Directories hold the main metadata components of the package you are building. These components will be deployed as an unmanaged package during development, but installed as a managed or unlocked package in production.

Pre-Install Directories are for unmanaged metadata components that should always be deployed prior to deploying or installing the package.

Post-Install Directories are for unmanaged metadata components that should always be deployed after deploying or installing the package.

Optional Configuration Directories are for additional sets of metadata components that will only be deployed on demand.

(Note that some projects may not have all of the types of directories and we should ideally only show the explanations for the ones that exist.)